src/app/screens/ConfirmPayment/index.tsx

```diff
@@ -1,6 +1,8 @@
+import "react-toastify/dist/ReactToastify.css";
```

Comment:
are those needed here? isn't this enough in the `/router/* files? (popup, prompt, options, welcome)

Reply:
The first time I tested it did not work without it here. Re-tested and it works. Prob a caching issue the first time. WIll remove.